src/block/NetherNylium.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\block;
+
+use pocketmine\block\utils\BlockEventHelper;
+use pocketmine\item\Item;
+
+class NetherNylium extends Opaque{
+
+	public function getDropsForCompatibleTool(Item $item) : array{
+		return [
+			VanillaBlocks::NETHERRACK()->asItem()
+		];
+	}
+
+	public function getSilkTouchDrops(Item $item) : array{
+		return [
+			$this->asItem()
+		];
+	}
+
+	public function isAffectedBySilkTouch() : bool{
+		return true;
+	}
+
+	public function ticksRandomly() : bool{
+		return true;
+	}
+
+	public function onRandomTick() : void{
+		$world = $this->position->getWorld();
+		$lightAbove = $world->getFullLightAt($this->position->getFloorX(), $this->position->getFloorY() + 1, $this->position->getFloorZ());
+		if($lightAbove < 4 && $world->getBlockAt($this->position->getFloorX(), $this->position->getFloorY() + 1, $this->position->getFloorZ())->getLightFilter() >= 2){
+			//nylium dies
+			BlockEventHelper::spread($this, VanillaBlocks::NETHERRACK(), $this);
```

Comment:
This is the wrong event maybe ?

Reply:
This is the code for grass, and nylium (to netheerack) have the same effect in vanilla..